access-common/src/main/java/edu/unc/lib/dl/ui/util/AnalyticsTrackerUtil.java

```diff
@@ -94,6 +124,13 @@ public void trackEvent(AnalyticsUserData userData, String category, String actio
         trackerThread.start();
     }
 
+    /**
+     * @param solrSearchService the solrSearchService to set
+     */
+    public void setSolrSearchService(SolrSearchService solrSearchService) {
+        this.solrSearchService = solrSearchService;
+    }
+
     public static class AnalyticsUserData {
         public String uip;
         public String cid;
```

Comment:
what does "cid" stand for? it might be helpful to use more self-documenting var names here, or to put a comment above each of these String vars to assist in comprehension of what this code is doing

Reply:
I've added comments, including a link to the google analytics documentation where these parameters are coming from